halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/NestableCommand.java

```diff
@@ -107,20 +107,23 @@ private void safeExecuteThis() {
               throw new RuntimeException("Unknown severity level " + severity);
           }
 
-          if (!remediation.isEmpty()) {
+          if (remediation != null && !remediation.isEmpty()) {
             AnsiUi.remediation(remediation);
           }
 
           // Newline between errors
           AnsiUi.raw("");
         }
+
+        System.exit(1);
       }
     } catch (Exception e) {
       if (GlobalOptions.getGlobalOptions().isDebug()) {
         throw e;
```

Comment:
debug can show the full stack that's fine. But I dont think the process should survive the method call. I could be wrong. I've seen fail fast in debug but not in production but never the other way around which is why I am pointing it out. I dont have a strong feeling about it.